src/database/aliasclients.c

```diff
@@ -259,8 +259,8 @@ void reset_aliasclient(sqlite3 *db, clientsData *client)
 	// Find corresponding alias-client (if any)
 	client->aliasclient_id = get_aliasclient_ID(db, client);
 
-	if(db_opened)
-		dbclose(&db);
+	// Close the database if we opened it here
+	if(db_opened) dbclose(&db);
```

Comment:
Changing it to be on one line is purely stylisic?

Reply:
Yes, it is made so a `grep dbclose` in the source code immediately reveals that this is not an unconditional call to this function.